makeflow/src/makeflow.c

```diff
@@ -89,7 +80,8 @@ static int makeflow_retry_flag = 0;
 static int makeflow_retry_max = MAX_REMOTE_JOBS_DEFAULT;
 
 static makeflow_gc_method_t makeflow_gc_method = MAKEFLOW_GC_NONE;
-static int makeflow_gc_param = -1;
+static uint64_t makeflow_gc_size   = 0;
```

Comment:
Now that you are looking at this code, please add comments to all these gc related variables.